packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -271,7 +273,7 @@ export async function transformGraphQLSchema(context, options) {
     buildParameters,
     projectDirectory: resourceDir,
     transformersFactory: transformerListFactory,
-    transformersFactoryArgs: [searchableTransformerFlag, storageConfig],
+    transformersFactoryArgs: { searchableTransformerFlag, storageConfig },
```

Comment:
I think we want `searchableTransformerFlag` to be `addSearchableTransformer` for better consistency with the first version of the transformer.